verification/curator-service/ui/src/components/AutomatedSourceForm.tsx

```diff
@@ -129,6 +141,7 @@ export default function AutomatedSourceForm(props: Props): JSX.Element {
                                 </div>
                                 <div className={classes.formSection}>
                                     <FastField
+                                        helperText="Required"
```

Comment:
UX nit: (can be addressed later) if all fields are required I think it's a better practice to say that in descriptive text at the top or similar, rather than marking each individual field as required.